Steps/PrGate.yml

```diff
@@ -133,9 +141,6 @@ steps:
     script: stuart_update -c ${{ parameters.build_file }} -p $(pkgs_to_build) -t ${{ parameters.build_targets}} -a ${{ parameters.build_archs}} TOOL_CHAIN_TAG=${{ parameters.tool_chain_tag}}
   condition: and(gt(variables.pkg_count, 0), succeeded())
 
-# Potential Extra steps

```

Comment:
There's nothing specifically bad about moving this, but was it fixing some existing use case or enabling a new one?

This alone could make this change a breaking change in my opinion, if, for example, the extra step assumed Stuart `setup` and `update` had already run.

Reply:
This is to resolve some steps needed to be run before the stuart_setup/stuart_update. For example, if the consumer would like to authenticate a nuget feed to complete the update step, the extra installation steps will need to be inserted here.

Generally, I think the extra_install_steps should happen after the generic install steps, which is where this change is located. But I agree that this is at risk of breaking existing pipelines at large. A layman question next: how do we track breaking changes/make that more obvious in this repo?

Reply:
If you're not seeing issues in testing we should be good then and mark is a breaking change. You can manually add the `impact:breaking-change` label.

Reply:
Thanks, I added the breaking-change tag and will update the other repos then test this again.

Reply:
All consumer repos are updated and testing in progress. For the status of consuming PRs, please check the links in https://github.com/microsoft/mu_devops/pull/154#issuecomment-1560077672.